index.html

```diff
@@ -9,21 +9,21 @@
     <title>Mourid Portfolio</title>
 </head>
 <body>
-        <header class="navbar">
+        <nav class="navbar">
         <div class="container">
             <div class="logo"><a href="#"><img src="img/My Logo.png"></a></div>
         <div class="menu"><img src="img/Union.svg" alt=""></div>
```

Comment:
*[OPTIONAL]*

:point_right:  ***Please, fix this before you merge***

- Hey, here's how you use the header in a better way to improve your semantic usage in your web pages. In this [lines of code](https://github.com/Aminocodex/Portfolio/blob/b83eb635fb1cf5776c0953e819dea2e9f611f311/index.html#L12-L17), you should use the header to wrap the introductory content in the nav like this below:

```
<header>
  <nav class="navbar">
    <div class="container">
      <div class="logo"><a href="#"><img src="img/My Logo.png"></a></div>
      <div class="menu"><img src="img/Union.svg" alt=""></div>
    </div>
  </nav>
</header>
```